tools/replay/ui.py

```diff
@@ -234,8 +237,8 @@ def ui_thread(addr, frame_address):
     SPACING = 25
 
     lines = [
-      info_font.render("ENABLED", True, GREEN if sm['controlsState'].enabled else BLACK),
-      info_font.render("BRAKE LIGHTS", True, RED if sm['carState'].brakeLights else BLACK),
+      info_font.render("ENABLED", True, GREEN if sm['controlsState'].enabled else WHITE),
```

Comment:
Let's leave the colors for enabled and brake. Now it's not immediately clear if white enable means enabled or not.